optimum/exporters/onnx/config.py

```diff
@@ -104,6 +106,7 @@ def inputs(self) -> Mapping[str, Mapping[int, str]]:
                 self.add_past_key_values(common_inputs, direction="inputs")
 
         if self._behavior is ConfigBehavior.DECODER:
+            # TODO: it is actually encoder_last_hidden_state
             common_inputs["encoder_outputs"] = {0: "batch_size", 1: "encoder_sequence_length"}
```

Comment:
It still is `encoder_outputs` when the `def inputs` mean `torch inputs for export`, but would mean `encoder_hidden_state` if considered ONNX inputs.

For me it's the former and the naming is correct. Maybe a comment would help to avoid confusion?